agenda-connectors-webapp/src/main/webapp/vue-app/agenda-connectors/exchange-connector/agendaExchangeConnector.js

```diff
@@ -61,5 +61,18 @@ export default {
         });
         return events;
       });
+  },
+  pushEvent(event){
```

Comment:
```suggestion
  pushEvent(event) {
```
also for getEvents method